pkg/apiserver/authentication/options/authenticate_options.go

```diff
@@ -58,6 +61,7 @@ func NewAuthenticateOptions() *AuthenticationOptions {
 		AuthenticateRateLimiterDuration: time.Minute * 30,
 		MaximumClockSkew:                10 * time.Second,
 		LoginHistoryRetentionPeriod:     time.Hour * 24 * 7,
+		LoginHistoryMaximumEntries:      48,
```

Comment:
Isn't 48 too small? Maybe we could increase it to, say 100?

Reply:
The default value has been modified to 100.